twisted/conch/ssh/channel.py

```diff
@@ -77,13 +77,19 @@ def __init__(self, localWindow = 0, localMaxPacket = 0,
 
 
     def __str__(self):
```

Comment:
Can you leave the __str__(self) method the way it is,
and introduce a new method called __bytes__(self), and explicitly call that method
instead of calling str()?

I have some patches to conch which do similar things.